sphinx_rtd_theme/layout.html

```diff
@@ -219,6 +219,23 @@
       });
   </script>
 
+  {# Do not conflict with RTD insertion of analytics script #}
+  {% if not READTHEDOCS %}
+    {% if theme_analytics_id %}
```

Comment:
Couldn't these two if statements be combined? It works but it does seem a bit awkward.

Reply:
Actually, the more I think about it while these can be combined they are sort of logically separate. I'm ok with it.